DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogic.cs

```diff
@@ -710,47 +653,10 @@ private static IEnumerable<DevicePropertyMetadata> GetDevicePropertyConfiguratio
             {
                 IsDisplayedForRegisteredDevices = true,
                 IsDisplayedForUnregisteredDevices = false,
-                IsEditable = false,
+                IsEditable = true,
```

Comment:
are you sure these are editable? doesn't make sense that location of the device is editable...